site2/docs/security-tls-authentication.md

```diff
@@ -44,10 +44,10 @@ $ openssl ca -config openssl.cnf -extensions usr_cert \
       -in admin.csr.pem -out admin.cert.pem
 ```
 
-This will give you a cert, `admin.cert.pem`, and a key, `admin.key-pk8.pem`. With `ca.cert.pem`, clients can used this cert and this key to authenticate themselves to brokers and proxies as the role token ``admin``.
+This command gives you a cert, `admin.cert.pem`, and a key, `admin.key-pk8.pem`. With `ca.cert.pem`, clients can used this cert and this key to authenticate themselves to brokers and proxies as the role token ``admin``.
```

Comment:
Use another word for "gives".
Reason: Don't attribute human qualities to software or hardware.
For guidelines, see https://developers.google.com/style/anthropomorphism